ui/admin/tests/helpers/selectors.js

```diff
@@ -15,4 +15,10 @@ export const TABLE_FIRST_ROW_RESOURCE_LINK = 'tbody tr:first-child a';
 export const RESOURCE_NOT_FOUND_SUBTITLE = '.rose-message-subtitle';
 export const RESOURCE_NOT_FOUND_VALUE = 'Error 404';
 
+export const PAGE_MESSAGE_DESCRIPTION = '.rose-message-description';
+export const PAGE_MESSAGE_LINK = '.rose-message-body .hds-link-standalone';
+
 export const HREF = (url) => `[href="${url}"]`;
+
+export const SIDEBAR_NAV_NTH_LINK = (i) =>
+  `[title="General"] a:nth-of-type(${i})`;
```

Comment:
For this selector, we could create a selector for FIRST, SECOND, THIRD, etc. However, I felt this is a simpler way to use it. Thoughts?

Reply:
For something as important as the sidenav, main navigation element of the app, I will aim for more specific selector. I.E: `SIDEBAR_ORGS`, `SIDEBAR_STORAGE_BUCKETS`.

My thinking:
- Easier to read/understand. I.E: to know what section we where clicking in each test, I had to open the UI, check I had the same flags and version as the test and then check what section belongs to the index 2 or 3. 
- The sidebar, despite not often, can suffer changes, therefore we will need to refactor all these indexes and that would not be easy for the reason I mentioned above.
- I am not saying it can't be a dynamic option such as a function, but if you want to go that route I will suggest use descriptive names and not indexes. I.E: `.dom(commonSelectors.SIDEBAR_NAV_LINK('storage_buckets'))`.


Happy to discuss this at tech time 😃

Reply:
As discussed within our 1:1 I will wait for changes within this function 😉

Reply:
This has been updated. Please take another look when you have a chance.